docs/host_binaries_solutions_comparison.md

```diff
@@ -0,0 +1,262 @@
+# Host binaries solutions comparison
+
+## tl;dr
+
+After research and test git-annex, we think it's not suitable for our purposes. We try to use GitHub Releases, documentation and automating process.
```

Comment:
Conclusions / status shoud be on the bottom of document.

Reply:
Thanks, changed!